.github/workflows/build-docs.yml

```diff
@@ -104,11 +106,11 @@ jobs:
         env:
           GITHUB_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
 
+      # Commits to pull requests result in preview deployments
       - name: Publish Preview to Vercel
         uses: amondnet/vercel-action@v25
         if: |
-          github.event_name == 'pull_request' ||
-          (github.event_name == 'push' && (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release-') || startsWith(github.ref, 'refs/heads/docs-prod')))
+          github.event_name == 'pull_request'
```

Comment:
@neverett please check my sanity on these `if` conditionals.

Reply:
This one will run on all PRs, is that right?